Assets/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "com.microsoft.mixedrealitytoolkit",
+  "version": "2.0.0",
+  "displayName": "Mixed Reality Toolkit",
+  "description": "MRTK virtal package.",
+  "unity": "2018.4",
+  "unityRelease": "0f1",
+  "dependencies": {
+ },
+ "keywords": [
+    "holotoolkit",
+	"holotoolkit-unity",
+	"mixed-reality",
+	"hololens",
+	"mixedrealitytoolkit",
+	"mrtk",
+	"mixedrealitytookit-unity",
+	"openvr"
+  ],
+  "author": {
+    "name": "Microsoft",
+    "email": "opencode@microsoft.com",
```

Comment:
I wonder if this should be an MRTK team email address?

Reply:
Yes, I agree. I reached out to @wiwei on this yesterday and asked what e-mail to use. He said there wasn't a public one for MRTK but he was going to bring it up in an internal thread. I'd be happy to change this to whatever the team feels it should be.

Reply:
Turns out e-mail address is optional. I missed that in the schema definition. I just removed the address altogether.